org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -169,7 +169,7 @@ abstract class AbstractLanguageServerTest implements Consumer<PublishDiagnostics
 
     protected def dispatch String toExpectation(SymbolInformation it) '''
         symbol "�name�" {
-            kind: �kind�
+            kind: �kind.value�
```

Comment:
The test expectation would probably be more readable if we were using the symbol names. But we can change that once someone is at it and wants to improve on that.